Testing/SystemTests/tests/analysis/WeightedLeastSquaresTest.py

```diff
@@ -118,27 +118,6 @@ def setUp(self):
         if not self.__class__.workspace:
             self.__class__.workspace = load_fitting_test_file_ascii(self.filename)
 
-    def test_misses_expected_solution(self):
```

Comment:
The fit doesn't converge on all platforms and gives a result that nobody would want to use for anything.